### ISSUE TITLE
Audio is gone

### ISSUE THREAD

**Issue by rica01** [TheGoddessInari/rambox#355]
Hello.

The sound from my Hamsket installation is gone. The program does not show in Windows Mixer and I don't know what to do. I am running the latest version on Win 11 64bits

**Comment by emmegamma** [comment]
Same same, but still on Win 10

**Comment by emmegamma** [comment]
Oh, well, it's apparently because of a Win update. It was enough to reinstall Hamsket

**Comment by tdpeuter** [comment]
Are you sure that you are not in `Don't disturb` mode?

**Comment by emmegamma** [comment]
Oh! I can't really exclude that now. I was certanly not expecting the "Don't disturb" mode to also disable audio from voice messages. I'd actually consider that a misfunctionality/misnaming: it's not "Don't disturb" mode (if I'm clicking on "play", it's not disturbing!), it's "Mute" mode then

**Comment by tdpeuter** [comment]
Yeah, don't disturb indeed acts more like a mute function. Worth keeping in mind if you ever wonder why audio isn't playing :)
